mods/stairs/init.lua

```diff
@@ -233,6 +235,121 @@ function stairs.register_slab(subname, recipeitem, groups, images, description,
 end
 
 
+-- Register corners
+-- Node will be called stairs:stairi_<subname>
+
+function stairs.register_stairi(subname, recipeitem, groups, images, description, sounds)
+	minetest.register_node(":stairs:stairi_" .. subname, {
```

Comment:
just a note, and heads up. the ":" before the mod name is actually in the current default stairs mod.
I repeated the mistake as I used copy/paste and did not proof read it close enough